planet/src/main/webapp/js/placesItinerary.js

```diff
@@ -13,56 +13,140 @@
 // limitations under the License.
 
 // This file contains functions related to the places part of the itinerary page
+
+import TimeRange from './TimeRange.js';
+
+// Declare global functions.
 window.openAddPlaceForm = openAddPlaceForm;
 window.closeAddPlaceForm = closeAddPlaceForm;
 
+// Declare global variables/constants.
 const database = firebase.database();
+let map = new google.maps.Map(document.getElementById('empty-map'));
+let savedPlaces;  // Object that contains all the places information
```

Comment:
nit: init this to empty object here too: {}

Reply:
also, maybe "displayedPlaces" is a more accurate variable name?

Reply:
Okay